### DISCUSSION TITLE
Reset / Clear Custom Trait

### DISCUSSION THREAD

**Discussion by michaelhofer-slg** [GrapesJS/grapesjs#6007] [Q&A]
We use a lot of custom traits and have noticed that there seems to be no way to clear or reset the custom traits.

The style manager has the option to reset values by clicking on the `x`:

![image](https://github.com/user-attachments/assets/fd9ffc6a-9091-4b26-b26e-52c88dda366f)

Is there a way to achieve the same behavior with custom traits? I have checked in the traits API documentation (https://grapesjs.com/docs/modules/Traits.html#events), but can't find a way to achieve the same result.

**Comment by RakulAgn** [comment]
@michaelhofer-slg you can add `x` to your trait whenever there is a value on your trait once clicked you can remove it i think for there don't need a api for this if i am right

**Comment by michaelhofer-slg** [comment]
Is there a way to enable the above mentioned reset feature on custom traits? I haven't found a solution for this yet.
